.bazelversion

```diff
@@ -1 +1 @@
-6.0.0
+6.1.0
```

Comment:
We found locally that we needed Bazel 6.1 to get the Bazel toolchain detection working with Xcode 14.3 on Intel machines.

Reply:
yeah IIRC to pick up [this](https://github.com/bazelbuild/bazel//commit/43dadb275b3f9690242bf2d94a0757c721d231a9) right?

Reply:
Yep, I believe it's in 6.1.**1**

Reply:
Thoughts on splitting these PRs? We should probably bump to 6.1.1, tag a release then merge this and tag again